Dockerfile

```diff
@@ -398,6 +398,10 @@ WORKDIR ${AIRFLOW_SOURCES}
 
 ENV PATH="${HOME}:${PATH}"
 
+# Disable writing .pyc files - slightly slower imports but not messing around when switching
+# Python version and avoids problems with root-owned .pyc files in host
+ENV PYTHONDONTWRITEBYTECODE="true"
```

Comment:
Since this is only designed for "local" mode could we set this as an env var in the docker-compose file(s) instead of here? (I can already tell people are going to try and take these CI images and run them before we have a "production" one ready)

Reply:
Sure. I would have to move it to all scripts then because we use those scripts in_container as entrypoints (so the test entrypoint will not be used). But since we have now _in_container_utils.sh source everywhere it will be easy.

Reply:
It can't live in the `environment:` section of the compose file?

Reply:
Not only. Docker-compose is used only to run environment for tests. Then for pylint tests, docs building etc we use pure docker (we need much smaller slim image and we do not need any of the other containers like postgres etc.) . We could set the environment in both docker-compose and in docker instructions. But yeah. You are right that setting it in the entrypoint script is not a good idea as well :). I will move it to the "host" scripts.